README.rst

```diff
@@ -151,6 +150,16 @@ dynamic programming and is linear O(n):
     >>> print(big_o.big_o(fib_dp, big_o.datagen.n_, n_repeats=100, min_n=200, max_n=1000)[0])
     Linear: time = -1.8E-06 + 7.3E-06*n (sec)
 
+Report Generation
+-------
+
+We can generate the a report using the output generated from calling the big_o
+function. The report defines the best time complexity along with the the others 
+estimates. The results are sorted based on the residuals and the rankings are 
+put in place to allow comparisons.
```

Comment:
I'm not sure what you mean by "the rankings are put into place", could you please rephrase?